### PR TITLE
arcadedb: Add version 23.6.1

### LINKED ISSUE #11638: [Request]: ArcadeDB
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

arcadedb

### Description

The Next Generation Multi-Model Database

### Homepage

https://arcadedb.com

### Download Link(s)

https://github.com/ArcadeData/arcadedb/releases

### Some Indication of Popularity/Repute

ArcadeDB is a successor to OrientDB (owned by SAP now), the same creator, OSS development model, fundamental rewrite, high-performance, high-availability

arcadedb-stars

### PR DESCRIPTION
**Pull request by littleli**
This PR adds ArcadeDB, an OSS successor to OrientDB, multimodal high-performance database with HA capabilities.

Closes #11638

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

Comments on manifest

ArcadeDB is written in Java and comes with a structure that's similar to a lot of Java projects.
It contains two scripts `console.bat` and `server.bat` that are unfortunately very generic names, that's why I decided to come up with two shims with `arcadedb-` prefix. Also set `ARCADEDB_HOME` environment variable is a huge convenience required by launch scripts and position of following folders.
Folders `config`, `database` and `log` are persisted so they don't disapper between updates.

### CODE REVIEW

File: bucket/arcadedb.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "version": "23.6.1",
+    "description": "Multi-model database, one DBMS that supports SQL, Cypher, Gremlin, HTTP/JSON, MongoDB and Redis.",
+    "homepage": "https://arcadedb.com",
+    "license": "Apache-2.0",
+    "notes": [
+        "After an installation or update please close all your terminal sessions before interacting with ArcadeDB commands.",
+        "Available commands:",
+        "  arcadedb-console",
+        "  arcadedb-server"
+    ],
```
Comment (HUMORCE) [inline bucket/arcadedb.json:11]:
```suggestion
```

- With Current junction, seem we does not need to close the terminal session after install/update
- `scoop info <app>` will list commands/executable file/binaries

File: bucket/arcadedb.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+    "version": "23.6.1",
+    "description": "Multi-model database, one DBMS that supports SQL, Cypher, Gremlin, HTTP/JSON, MongoDB and Redis.",
```
Comment (HUMORCE) [inline bucket/arcadedb.json:3]:
```suggestion
    "description": "Multi-model database, one DBMS that supports SQL, Cypher, Gremlin, HTTP/JSON, MongoDB and Redis",
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/arcadedb.json (hunk 1 above)
Comment (littleli) [reply bucket/arcadedb.json:11]: Ok. Good to know! Thanks

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**